pkg/arvo/sys/vane/gall.hoon

```diff
@@ -2,1457 +2,2882 @@
-  $:  misvale/misvale-data                              ::  bad reqs
-      vel/worm                                          ::  cache
-      arms=(map [term path] (unit (pair @ud term)))     ::  ap-find cache
-      mom/duct                                          ::  control duct
-      liv/?                                             ::  unstopped
-      toc/torc                                          ::  privilege
-      tyc/stic                                          ::  statistics
-      ged/gest                                          ::  subscribers
-      hav/vase                                          ::  running state
-      byk/beak                                          ::  update control
-      pyl/(map bone mark)                               ::  req'd translations
-      zam/scar                                          ::  opaque ducts
-  ==                                                    ::
-++  sofa                                                ::  queue for blocked
-  $:  kys/(qeu (trel duct prey club))                   ::  queued kisses
-  ==                                                    ::
-++  stic                                                ::  statistics
-  $:  act/@ud                                           ::  change number
-      eny/@uvJ                                          ::  entropy
-      lat/@da                                           ::  time
-  ==                                                    ::
---                                                      ::
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::  vane header
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::
+::  (rest of arvo)
+::
+|%
+::
+::  +internal-gift: synonym for +cuft.
+::
+++  internal-gift  cuft
+::
+::  +internal-task: synonym for +cush.
+::
+++  internal-task  cush
+::
+::  +agent-action: synonym for +club.
+::
+++  agent-action  club
+::
+::  +coke: cook.
+::
+++  coke
+  $?  %inn
+      %out
+      %cay
+  ==
+::
+::  +volt: voltage.
+::
+++  volt  ?(%low %high)
+::
+::  +security-control: security control.
+::
+++  security-control  $@(?(%iron %gold) [%lead p=ship])
+::
+::  +reverse-ames: reverse ames message.
+::
+++  reverse-ames
+  $%
+      :: diff
+      ::
+      [action=%d p=mark q=*]
+      ::  etc.
+      ::
+      [action=%x ~]
+  ==
+::
+::  +forward-ames: forward ames message.
+::
+++  forward-ames
+  $%
+      :: message
+      ::
+      [action=%m mark=mark noun=*]
+      :: "peel" subscribe
+      ::
+      [action=%l mark=mark path=path]
+      :: subscribe
+      ::
+      [action=%s path=path]
+      :: cancel+unsubscribe
+      ::
+      [action=%u ~]
+  ==
+::
+::  +foreign-response: foreign response.
+::
+++  foreign-response
+  $?  %peer
+      %peel
+      %poke
+      %pull
+  ==
+--
+::
+::  (local arvo)
+::
+|%
+::
+::  +internal-note: +ap note.
+::
+++  internal-note
+  $%  [task=%meta =term =vase]
```

Comment:
It's interesting you put a face on the tag.  I haven't seen this before.  The usual idiom is to `?-  -.note`, which I consider a good use of lark.  The fact that the tag is in the head is actually important to the compiler, so that lark can be thought of as a standard name.  Otherwise everybody has to define their own `task` face or whatever, which is less immediately grokkable.